client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/GetOperation.java

```diff
@@ -27,7 +27,8 @@
    public GetOperation(Codec codec, ChannelFactory channelFactory,
                        Object key, byte[] keyBytes, byte[] cacheName, AtomicInteger topologyId, int flags,
                        Configuration cfg, DataFormat dataFormat, ClientStatistics clientStatistics) {
-      super(GET_REQUEST, GET_RESPONSE, codec, channelFactory, key, keyBytes, cacheName, topologyId, flags, cfg, dataFormat, clientStatistics);
+      super(GET_REQUEST, GET_RESPONSE, codec, channelFactory, key, keyBytes, cacheName, topologyId, flags, cfg,
+            dataFormat, clientStatistics, null);
```

Comment:
It seems like you are not adding telemetry to read operations. Why ?

Reply:
In this first version of tracing implementation I would avoid to trace all read operations. In the next version I would add a tracing filter to allow the user to set up what to trace and what not.